package-res/resources/web/prompting/PromptPanel.js

```diff
@@ -677,7 +679,8 @@ define(['cdf/lib/Base', 'cdf/Logger', 'dojo/number', 'dojo/i18n', 'common-ui/uti
          *
          * @name PromptPanel#_submit
          * @method
-         * @param {Object} options
+         * @param {Object}  [options]        Aditional configuration options.
+         * @param {Boolean} [options.isInit] Flag indicating if submit is being executed during initialization.
```

Comment:
A couple of times an object is provided with a property isInit. In the [PromptPanel](https://github.com/pentaho/pentaho-platform-plugin-common-ui/blob/master/package-res/resources/web/prompting/PromptPanel.js#L1289) and [ReportViewer](https://github.com/pentaho/pentaho-platform-plugin-reporting/blob/master/package-res/reportviewer/reportviewer.js#L670). [PIR](https://github.com/pentaho/pentaho-platform-plugin-interactive-reporting/blob/master/package-res/resources/web/pir/view.js#L5369) doesn't provide the options parameter so I documented it as optional. I'm not sure if it's always the only property.